lib/Stats.tsx

```diff
@@ -2,8 +2,19 @@ import React, { useCallback, useEffect, useState } from 'react'
 import styled from 'styled-components'
 import { useInterval } from 'react-hooks-shareable'
 
-import { Format, VideoProperties } from './PlaybackArea'
+import { Format, PlayerPipeline, VideoProperties } from './PlaybackArea'
 import { StreamStats } from './img'
+import { Html5VideoPipeline } from 'media-stream-library'
+
+const isHtml5VideoPipeline = (
+  pipeline: PlayerPipeline | null | undefined,
+): pipeline is Html5VideoPipeline => {
+  return (
+    pipeline !== undefined &&
+    pipeline !== null &&
+    (pipeline as Html5VideoPipeline).tracks !== undefined
```

Comment:
Could perhaps be simplified with some optional chaining. There might be one or more other places in this review as well.